### PR TITLE
Update deployment instructions.

### PR DESCRIPTION
**Pull request by naved001**
I was able to do a successful ironic deployment using tripleo on a VM and on a baremetal server with the instructions in this patch.

* Update the NeutronMechanismDrivers
* update references to environment files to include neutron-ovn-standalone and neutron-ml2-ansible.
* Put a sample `standlaone_parameters.yaml` file.

I'll add instructions on how to enable/configure ansible-networking in a separate PR (because it's still a work in progress).

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -23,7 +32,33 @@ resource_registry:
...
+  - ipmitool
+  IronicEnabledManagementInterfaces:
+  - ipmitool
+  IronicCleaningDiskErase: 'metadata'
+  IronicInspectorSubnets:
+  - ip_range: 192.168.1.200,192.168.1.250
+  IronicInspectorInterface: 'br-ctlplane'
+```
+
+A sample `standalone_parameters.yaml` file with all other parameters looks like this.
```
Comment (larsks) [inline docs/deployment.md]: It's not clear if the "this" in "...looks like this" is referring to the preceding or following content. Maybe clarify?  E.g., "...looks like the following:".

File: docs/deployment.md [hunk 2]
```diff
@@ -23,7 +32,33 @@ resource_registry:
...
+Note that `NeutronMechanismDrivers` is set to `ovn` and `baremetal`.
```
Comment (larsks) [inline docs/deployment.md]: It's probably worth mentioning what this parameter does and/or why these particular values are important.

File: docs/deployment.md [hunk 3]
```diff
@@ -12,6 +12,15 @@ ESI requires Ironic code that is currently only present in the master branch. Fo
 sudo -E tripleo-repos current-tripleo-dev
 ```
 
+and then install the tripleoclient and generate a file with the default ContainerImagePrepare value
```
Comment (larsks) [inline docs/deployment.md]: I would separate the two steps:

```
Next, install `tripleoclient`:

[...]

And finally, generate a container image configuration by running:

[...]
```

File: docs/deployment.md [hunk 4]
```diff
@@ -23,7 +32,33 @@ resource_registry:
   OS::TripleO::Services::NovaCompute: OS::Heat::None
 
 parameter_defaults:
-  NeutronMechanismDrivers: [openvswitch, baremetal]
+  NeutronMechanismDrivers: [ovn, baremetal]
```
Comment (tzumainn) [inline docs/deployment.md]: I don't think we can recommend ovn, now that we know that neutron dhcp won't work with it.

**Pull request closed without merging**